modules/ROOT/pages/7.1.1-release-notes.adoc

```diff
@@ -71,6 +71,33 @@ The {productname} {release-version} release includes an accompanying release of
 
 For information on the **<Premium plugin name 1>** plugin, see: xref:<plugincode>.adoc[<Premium plugin name 1>].
 
+=== Enhanced Tables
+
+The {productname} {release-version} release includes an accompanying release of the **Enhanced Tables** premium plugin.
+
+This **Enhanced Tables** plugin release includes the following fix.
+
+=== The sort algorithm would sometimes incorrectly compare words that start with a digit.
+// #TINY-10421
+
+Previously in **Enhanced Tables**, when trying to sort a table where the cells contained text that started with numbers followed by words, the table would only consider the number part of the text when sorting. This would cause the table to incorrectly sort the words.
+
+For example, when trying to sort the following table by column ascending (A-Z):
+
+[cols="1", options="header"]
+|===
+| Column 1
+
+| 3banana
+| 3apple
+|===
+
+The table would not sort them correctly due to it only considering the number part, so `3banana` would remain above `3apple`, even though `3apple` should come before `3banana` alphanumerically.
```

Comment:
```suggestion
As a consequence, the table would not sort them correctly due to it only considering the number part, so `3banana` would remain above `3apple`, even though `3apple` should come before `3banana` alphanumerically.
```

Reply:
Adding "As a consequence" doesn't read right here since this sentence is a continuation of the statement on line 85 just before the example table. I'll close this conversation but feel free to re-open it if you disagree.

Reply:
@kemister85 Adding "As a consequence" doesn't read right here since this sentence is a continuation of the statement on line 85 just before the example table. I'll close this conversation but feel free to re-open it if you disagree.